app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
+    redirect_to articles_path(@article)
+  end
+
+  def destroy
+    @article = Article.find(params[:article_id])
```

Comment:
create a `before_action` filter with name of `find_article` and put that line in that method